tests/unit/test_charm.py

```diff
@@ -52,7 +51,7 @@ def test_charm_reaches_active_status(urlopen_mock):
     act: trigger a configuration change with the required configs.
     assert: the charm reaches ActiveStatus.
     """
-    with open("tests/unit/files/metadata_1.xml", "rb") as metadata:
+    with open("tests/unit/files/metadata_unsigned.xml", "rb") as metadata:
```

Comment:
would `https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text` work for these?

Reply:
What's the difference between both?

Reply:
instead of needing to use a context manager you just read the file contents, there is also a read_binary or something like that